content/pages/download.md

```diff
@@ -3,26 +3,34 @@ title: Download
 template: pages/download
 versions:
   stable:
-    name: 2.4.2
-    release_announcement: /news/2024-11-27-mixxx-2-4-2-released/
-    download_manifest: https://downloads.mixxx.org/releases/2.4.2/manifest.json
+    name: 2.5.0
+    release_announcement: /news/2024-12-24-mixxx-2_5-released/
+    download_manifest: https://downloads.mixxx.org/releases/2.5.0/manifest.json
     downloads:
       - slug: windows
-        os: Windows 7 or later
-        packages:
-          - slug: win64
-            name: 64-Bit
+        os: Windows 10 build 1809 or later
+        text: |
+          Due to signing issues, we can`t provide a signed 2.5.0 release now. In the meantime please use 2.5-beta-152 which uses already the final source code of Mixxx 2.5.0..
+
+          [Download](https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi)
+
+          [2.5-beta-152-g70d7999549](https://github.com/mixxxdj/mixxx/commit/70d799954931e0cbd5db2404a598c941941ce2d2)
+
+          [sha256sum](https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi.sha256sum)
+
+          [Build Date Sat 21 December 2024 16:44](https://github.com/mixxxdj/mixxx/actions/runs/12446735096)
```

Comment:
This is not particularly pretty. I'm afraid that non tech-savvy windows users won't know what link to click. It would be nice we could still provide a proper download button to the snapshot build so users know what to click:
```suggestion
        text: |
          Due to signing issues, we can`t provide a signed 2.5.0 release now. In the meantime please use 2.5-beta-152 which uses already the final source code of Mixxx 2.5.0..

          [2.5-beta-152-g70d7999549](https://github.com/mixxxdj/mixxx/commit/70d799954931e0cbd5db2404a598c941941ce2d2)

          [sha256sum](https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi.sha256sum)

          [Build Date Sat 21 December 2024 16:44](https://github.com/mixxxdj/mixxx/actions/runs/12446735096)

        packages:
          - slug: win64
            name: Windows
            file_url: https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi
```